workflows/docker-data-science/docker/requirements.txt

```diff
@@ -0,0 +1,5 @@
+numpy
```

Comment:
we can pin all these packages to their latest versions, so that we always run the exact same code